ext/datasource/helper.go

```diff
@@ -211,31 +186,9 @@ func HotSpotParamRulesJsonConverter(src []byte) (interface{}, error) {
 	}
```

Comment:
In order to keep naming consistent.
Please rename function name, such as:HotSpotParamRuleJsonArrayParser.